internal/nostr/nostr.go

```diff
@@ -277,6 +277,59 @@ func (svc *Service) NIP47Handler(c echo.Context) error {
 	return c.JSON(http.StatusOK, event)
 }
 
+func (svc *Service) NIP47SubscriptionHandler(c echo.Context) error {
+	var requestData NIP47SubscriptionRequest
+
+	// send in a pubkey and authenticate by signing
+	if err := c.Bind(&requestData); err != nil {
+		return c.JSON(http.StatusBadRequest, ErrorResponse{
+			Message: fmt.Sprintf("error decoding subscription request: %s", err.Error()),
+		})
+	}
+
+	if (requestData.WebhookUrl == "") {
+		return c.JSON(http.StatusBadRequest, ErrorResponse{
+			Message: "webhook url is empty",
+		})
+	}
+
+	subscription := Subscription{
+		RelayUrl:   requestData.RelayUrl,
+		WebhookUrl: requestData.WebhookUrl,
+		Open:       true,
+		Authors:    &[]string{requestData.WalletPubkey},
+		Kinds:      &[]int{23195},
```

Comment:
this is for nip-47 notifications, right? it should be 23196

Reply:
ping